packages/circuit-ui/components/TopNavigation/TopNavigation.tsx

```diff
@@ -103,15 +105,16 @@ export function TopNavigation({
   profileIsActive,
   links,
   hamburger,
+  isLoading,
   ...props
 }: TopNavigationProps): JSX.Element {
   return (
     <Header role="banner" {...props}>
-      <Wrapper>
+      <div css={wrapperStyles}>
```

Comment:
With tailwind:

```suggestion
      <div className="flex items-stretch">
```

(it's in these sort of cases that utility classes really shine)

Reply:
Eh, I don't like Tailwind because I don't like adding a class for every CSS rule. I will admit that it can be useful for common flexbox styles though.